source/reference/minio-mc-admin/mc-admin-replicate.rst

```diff
@@ -198,6 +215,20 @@ Syntax
 
       The deployment ID can be found by running :mc-cmd:`mc admin replicate info ALIAS`
 
+   .. mc-cmd:: --disable-ilm-expiry-replication
+      :optional:
+
+      .. versionadded:: mc RELEASE.2023-12-02T02-03-28Z
+
+      Stops the replication of ILM expiration rules between peer sites.
```

Comment:
Wonder if we need a note here that this does *not* remove any ILM expiration rules already synchronized between sites.

Reply:
Added a sentence to this effect.